app/views/logs/delete_duplicates.html.erb

```diff
@@ -0,0 +1,39 @@
+<% content_for :before_content do %>
+  <% content_for :title, "Are you sure you want to delete #{@duplicate_logs.count == 1 ? 'this duplicate log' : 'these duplicate logs'}?" %>
+  <%= govuk_back_link href: @log.lettings? ? lettings_log_path(@log) : sales_log_path(@log) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-l">
+      <%= content_for(:title) %>
+    </h1>
+
+    <%= govuk_warning_text(text: "You will not be able to undo this action.") %>
+
+    <p class="govuk-body govuk-!-margin-bottom-2">
+      <%= @duplicate_logs.count == 1 ? "This log" : "These logs" %> will be deleted:
+    </p>
+    <ul class="govuk-list">
+      <% @duplicate_logs.each do |duplicate_log| %>
+        <li>
+          <strong>
+            <%= govuk_link_to "Log #{duplicate_log.id}", duplicate_log.lettings? ? lettings_log_path(duplicate_log) : sales_log_path(duplicate_log) %>
+          </strong>
+        </li>
+      <% end %>
+    </ul>
+
+    <div class="govuk-button-group">
+      <%= govuk_button_to @duplicate_logs.count == 1 ? "Delete this log" : "Delete these logs",
```

Comment:
Does this work?
```suggestion
      <%= govuk_button_to "Delete #{"this".pluralize(@duplicate_logs.count)} #{"log".pluralize(@duplicate_logs.count)}",
```
or maybe even with "this log".pluralize
I suspect the first works and the second doesn't, first is them similarly verbose so maybe not better

Reply:
Doesn't work unfortunately

Reply:
You could add:

```ruby
ActiveSupport::Inflector.inflections(:en) do |inflect|
    inflect.irregular 'this', 'these'
end
```

to https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/config/initializers/inflections.rb, then it would work

Reply:
Yes! Good point!
I'm not gonna do it tho, I don't think double interpolation is more readable in this case, but shout if you disagree